### ISSUE TITLE
Disable WPF Listview Column Resizing

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/wpf#9412]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Disable-WPF-Listview-Column-Resizing/10217408)._

---
Add a property to prevent resizing of a column in a WPF Listview but still make it sortable. 

There is one way around this, but it disables sorting on that column.

---
#### Feedback Bot on 12/1/2022, 08:40 AM: 

(private comment, text removed)

**Comment by miloush** [comment]
I don't think that is a good practice and some effort should be required on the developer side to enforce this.

The effort is pretty minimal anyway:

```C#
public class NonResizableGridViewColumnHeader : GridViewColumnHeader
{
    public override void OnApplyTemplate()
    {
        if (Role != GridViewColumnHeaderRole.Normal)
            base.OnApplyTemplate();
    }
}
```
